Use alsa-lib instead of tinyalsa

shakthi-prashanth-m opened the pull request:
Our goal is to redirect Android audio traffic to remote pulse audio server via pulse alsa plugin. This leaves routing, mixing, etc to Pulse audio server.

Commit:
Use alsa-lib instead of tinyalsa

Redirect audio hal traffic to remote pulse audio on Host via pulse
audio alsa plugin.

Tracked-On:
Signed-off-by: Prashanth M, Shakthi <shakthi.prashanth.m@intel.com>

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

FAILURE: CheckBug Bad comments/Bugs

For more information, see: /absp/builders/celadon-autobuild/builds/1465

Commit:
Revert "Use alsa-lib instead of tinyalsa"

This reverts commit e89b31fdf5ce84f94ef813bce059b3d16e2b8a54.

Commit:
Fix klockwork issue

Socket endpoint is closed while audio stream is closed.

Tracked-On: https://jira.devtools.intel.com/browse/OAM-88926
Signed-off-by: Prashanth M, Shakthi <shakthi.prashanth.m@intel.com>

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

FAILURE: CheckBug Bad comments/Bugs

For more information, see: /absp/builders/celadon-autobuild/builds/1501
